src/pages/BookSubmissionForm.vue

```diff
@@ -196,7 +196,6 @@ export default {
     },
     clearSubmission(si) {
       // remove submission and existing book
-      // eslint-disable-next-line fp/no-delete
```

Comment:
If disabling the linter is no longer needed, then it would suggest that `fp/no-delete` is not being enforced. The intention is that we do want to enforce `fp` linting such as `fp/no-delete`, `fp/no-class`, and `fp/no-loops`. Maybe something changed with the eslint config so that the default `fp` rules are not included?